django/db/models/sql/constants.py

```diff
@@ -35,3 +35,8 @@
 # SQL join types.
 INNER = 'INNER JOIN'
 LOUTER = 'LEFT OUTER JOIN'
+
+# Set operators
+UNION = 'union'
+INTERSECT = 'intersect'
+EXCEPT = 'except'
```

Comment:
If you change `set_operators` in Oracle `DatabaseWrapper` to:
```python
set_operators = {
    'union': 'UNION',
    'intersect': 'INTERSECT',
    'except': 'MINUS',
} 
```
then everything should work. Furthermore all related feature flags can be set to `True` in the Oracle backend.